pkg/kritis/admission/admission.go

```diff
@@ -154,31 +261,3 @@ func checkBreakglass(pod *v1.Pod) bool {
 func metadataClient() (metadata.MetadataFetcher, error) {
 	return containeranalysis.NewContainerAnalysisClient()
 }
-
-func returnStatus(status constants.Status, message string, w http.ResponseWriter) {
-	response := &v1beta1.AdmissionResponse{
-		Allowed: (status == constants.SuccessStatus),
-		Result: &metav1.Status{
-			Status:  string(status),
-			Message: message,
-		},
-	}
-	if err := writeHttpResponse(response, w); err != nil {
-		glog.Error("error writing response:", err)
-	}
-}
-
-func writeHttpResponse(response *v1beta1.AdmissionResponse, w http.ResponseWriter) error {
```

Comment:
umm you dont need this?

Reply:
no, this is done at the end of the main review function